packages/account-v2/src/components/base/WalletText/WalletText.scss

```diff
@@ -0,0 +1,184 @@
+$color-map: (
+    general: #333333,
+    primary: #999999,
+    success: #4bb4b3,
+    warning: #ffad3a,
+    error: #ec3f3f,
+    white: #ffffff,
+    black: #000000,
+    less-prominent: #999999,
+    red: #ff444f,
+    blue: #377cfc,
+    green: #17eabd,
+    orange: #ff9c13,
+    system-dark-2-general-text: #c2c2c2,
+);
```

Comment:
Don't copy paste files from Wallets
If you want to reuse then tweak it
i think in v2 you have to use tailwind so stick to only one

Reply:
@amina-deriv can u comment on this please

Reply:
I guess it makes sense tho to define them in tailwind config